EtherealEngine/etherealengine
# @Swc/core update to newer version

## Summary

- [x] Updated `@swc/core` from 1.3.41 to 1.3.96
- Added `@swc/counter: ^0.1.2`
- And `@swc/types: ^0.1.5`
- Removal of empty lines in package.json files due to the custom script I ran to modify the
`@swc/core` version

```
ee-bot and ee-develoment-test-suite also require @swc/core version update
```

## References
closes #9116 

## QA Steps
✔️ Ran `npm run check-errors`, no issues found
✔️ Ran `npm run check`, also no issues
✔️ Ran dev, everything seems functional
